CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 - Make plugin Gradle 8 compatible ([#428](https://github.com/getsentry/sentry-android-gradle-plugin/pull/428))
 - Fix `uploadSentryNativeSymbols` task for Gradle 8 ([#447](https://github.com/getsentry/sentry-android-gradle-plugin/pull/447))
 
+- Write enabled instrumentations and proguard uuid to manifest ([#441](https://github.com/getsentry/sentry-android-gradle-plugin/pull/441))
```

Comment:
We should move this out of the released version to the Unreleased section (not sure why Danger has not checked it, probably we don't have it on this repo)